### PR TITLE
Add Solana docs

### PR DESCRIPTION
**Pull request by akegaviar**

### CODE REVIEW

File: docs/operations/solana/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    content: Learn what Solana networks you can deploy your nodes on and how to connect to your Solana node.
+  - name: keywords
+    content: solana node deploy program
+---
+
+# Introduction
+
+This section outlines the fundamentals of and operations for Solana.
+
+Deploying a Solana node with Chainstack provides you with a near-instantly fully synchronized Solana node through the use of [Bolt](/glossary/bolt).
```
Comment (GranderStark) [inline docs/operations/solana/README.md:13]: We are not using Bolt for Solana

File: docs/operations/solana/tools.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+```
+
+</template>
+</CodeSwitcher>
+
+::: tip See also
+
+* [Creating a token and vesting the token in your program](/tutorials/solana/creating-a-token-and-vesting-the-token-in-your-program)
+
+:::
```
Comment (GranderStark) [inline docs/operations/solana/tools.md:172]: As I see we have covered WebSockets in other protocols, but we have not cover it here.
Currently we have it enabled in configuration of our node, it is responding. 
@akegaviar , I can provide some examples to test it out.

File: docs/operations/solana/types.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  - name: keywords
+    content: solana node shared dedicated blockchain
+---
+
+# Types
+
+Chainstack supports deploying a Solana node of the following types:
+
+* Shared — the node resources are shared with other platform users.
+* Dedicated — the node resources are exclusive to you.
```
Comment (GranderStark) [inline docs/operations/solana/types.md:14]: Current situation - we have disable Dedicated option for Solana

Comment (akegaviar) [reply docs/operations/solana/types.md:14]: Will the manual deployment option be available? Leave the dedicated in or remove? @easeev

Comment (easeev) [reply docs/operations/solana/types.md:14]: Remove for now, will add later

File: docs/operations/solana/README.md (hunk 1 above)
Comment (akegaviar) [reply docs/operations/solana/README.md]: Fixed

File: docs/operations/solana/types.md (hunk 3 above)
Comment (akegaviar) [reply docs/operations/solana/types.md]: Fixed

File: docs/operations/solana/tools.md (hunk 2 above)
Comment (akegaviar) [reply docs/operations/solana/tools.md:181]: I did a bit of testing, and in the solana tool it actually assumes/computes a wss endpoint from the https one, and does this incorrectly in our case. added a note on that.

**Review by GranderStark — APPROVED** [review]

**Pull request merged**